fdbserver/workloads/BackupCorrectness.actor.cpp

```diff
@@ -733,7 +731,7 @@ struct BackupAndRestoreCorrectnessWorkload : TestWorkload {
 						       restoreIndex,
 						       range.toString().c_str(),
 						       restoreTag.toString().c_str());
-						restores.push_back(backupAgent.restore(cx,
+						restores.push_back(backupAgent.restore(self->extraDB,
```

Comment:
Don't think we should use extraDB when encryption is off

Reply:
I think it make sense to make no distinction between encryption on/off, since the normal use case for backup is restoring to a new cluster.